en/views/helpers/form.rst

```diff
@@ -2444,7 +2444,7 @@ a multiple select input for belongs to many associations::
     echo $this->Form->control('tags._ids', [
         'type' => 'select',
         'multiple' => true,
-        'options' => $tagList,
+        'options' => $tags, // $tags is the output of $this->Articles->Tags->find('list')->all() in the controller
```

Comment:
Should we include ->toArray() here?

Reply:
I don't see any reason to do that, the ORM generates collections by default and the form helper can use them just fine.

Reply:
In fact people might incorrectly assume that the options must always be an array if we use `toArray()`.